<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitterWidget</class>
 <widget class="QWidget" name="SplitterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SplitterWidget</string>
  </property>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>420</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNom"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Texte à décoder :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDecoder"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDecoder">
      <property name="text">
       <string>Décoder</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Prénom :</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPrenom"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Âge :</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumberAge"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditAge"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Poids :</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPoids"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Taille :</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditTaille"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>SplitterWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
